packages/neo-one-client-core/src/__ledger_tests__/LedgerKeyStore.test.ts

```diff
@@ -0,0 +1,184 @@
+import { utils } from '@neo-one/utils';
```

Comment:
make `LedgerKeyStore` take a `LedgerHandler` to use (with a default set to the global `LedgerHandler`) and then pass in a mock for these tests. Move these tests to the regular unit test folder.

I.e. test `LedgerKeyStore` with mocks. If you're also testing `LedgerHandler` functionality here, then it should be moved to the `LedgerHandler` test.